definitions/checks/candlepin/validate_db.rb

```diff
@@ -11,7 +11,7 @@ class ValidateDb < ForemanMaintain::Check
 
     def run
       result, result_msg = feature(:candlepin_database).execute_cpdb_validate_cmd
-      next_steps = []
+      next_steps = [Procedures::StartService.new(:service => 'postgresql')]
```

Comment:
@iNecas,
Instead of hard-coding service name, should I need to handle this based on DB type, right?

Reply:
I guess, we can add this step later on after [PR-151](https://github.com/theforeman/foreman_maintain/pull/151) . This step i.e procedure helps to start respective DB service.

Reply:
:+1: for deferring this decision for now.